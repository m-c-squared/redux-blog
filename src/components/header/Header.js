import React from 'react';
import { Link } from 'react-router-dom';
import { ROUTES } from '../../routes/routes';


export default function Header() { 
  return (
    <header>
      <nav>
        <Link to={ROUTES.HOME.linkTo()}>Home</Link>
        <Link to={ROUTES.ALL_USERS.linkTo()}>All Users</Link>
        <Link to={ROUTES.USER_CONTAINER.linkTo()}>User Detail</Link>
        <Link to={ROUTES.ALL_COMMENTS.linkTo()}>All Comments</Link>
        <Link to={ROUTES.ALL_POSTS.linkTo()}>All Posts</Link>
      </nav>
    </header>
  );

}

