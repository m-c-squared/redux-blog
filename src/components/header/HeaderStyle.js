import styles from 'styled-components';
import { Link } from 'react-router-dom';

export const HeaderStyle = styles(Link)`
  display: inline;
  justify-content: space-around;
  font-size: 15px;
  padding-right: 20px;
`;
