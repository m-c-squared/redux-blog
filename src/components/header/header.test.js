import React from 'react';
import renderer from 'react-test-renderer';
import header from './Header';

describe('Header', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <header />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
