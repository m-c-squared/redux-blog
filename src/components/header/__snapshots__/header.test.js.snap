// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches a snapshot 1`] = `
<header
  className="sc-fjdhpX fGiJGr"
>
  <nav>
    <a
      className="sc-jTzLTM aTBmR"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="sc-jTzLTM aTBmR"
      href="/users"
      onClick={[Function]}
    >
      Users
    </a>
    <a
      className="sc-jTzLTM aTBmR"
      href="/comments"
      onClick={[Function]}
    >
      Comments
    </a>
    <a
      className="sc-jTzLTM aTBmR"
      href="/posts"
      onClick={[Function]}
    >
      Posts
    </a>
  </nav>
</header>
`;
